backend/clubs/views.py

```diff
@@ -2895,6 +2895,14 @@ def get(self, request):
         return Response({"count": ans})
 
     def post(self, request):
+        if settings.ZOOM_VERIFICATION_TOKEN is not None:
```

Comment:
You can make this check a bit more relaxed by removing the `is not None`, which also fails on empty string.